### PR TITLE
pdbedit: add page; smbpasswd: improve comments & add `sudo`

### PR DESCRIPTION
**Pull request by cyqsimon**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):** latest at time of writing (`Samba 4.15.2`)

### CODE REVIEW

File: pages/linux/pdbedit.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# pdbedit
+
+> Directly edit the Samba user database.
```
Comment (mfrw) [inline pages/linux/pdbedit.md:3]:
```suggestion
> Edit the Samba user database.
```

File: pages/linux/pdbedit.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+> Can do more fancy things than `smbpasswd`.
```
Comment (mfrw) [inline pages/linux/pdbedit.md:4]: Can we instead put a See also:
```suggestion
> Similar to `smbpasswd`.
```

File: pages/linux/pdbedit.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+> More information: <https://manned.org/pdbedit.8>.
```
Comment (mfrw) [inline pages/linux/pdbedit.md:5]:
```suggestion
> More information: <https://manned.org/pdbedit>.
```

File: pages/linux/smbpasswd.md [hunk 4]
```diff
@@ -1,20 +1,21 @@
 # smbpasswd
 
-> Change a user's SMB password.
-> Samba users must also have a local Unix account.
+> Add/remove a Samba user or change their SMB password.
+> Samba users must have an existing local Unix account.
+> More information: <https://manned.org/smbpasswd.8>.
```
Comment (mfrw) [inline pages/linux/smbpasswd.md:5]:
```suggestion
> More information: <https://manned.org/smbpasswd>.
```

**Review by mfrw — COMMENTED** [review]
Thank you for your contribution @cyqsimon 
Have a few review comments.

File: pages/linux/pdbedit.md (hunk 2 above)
Comment (cyqsimon) [reply pages/linux/pdbedit.md:4]: `pdbedit` provides a superset of features to `smbpasswd`. I'm okay with less casual language but then I'm not sure what's the best way to word it.

File: pages/linux/smbpasswd.md (hunk 4 above)
Comment (cyqsimon) [reply pages/linux/smbpasswd.md:5]: https://manned.org/smbpasswd.5 also exists. It not having `.8` okay?

Comment (mfrw) [reply pages/linux/smbpasswd.md:5]: Ahh :( sorry for this .. yep having an 8 is absolutely fine then

Comment (CleanMachine1) [reply pages/linux/smbpasswd.md:5]: we should use mfrw's suggestion.

File: pages/linux/pdbedit.md (hunk 2 above)
Comment (cyqsimon) [reply pages/linux/pdbedit.md]:
```
> For simple user add/remove/password, you can also use `smbpasswd`.
```
I suppose this is somewhat more descriptive.

File: pages/linux/pdbedit.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
+# pdbedit
+
+> Edit the Samba user database.
+> For simple user add/remove/password, you can also use `smbpasswd`.
+> More information: <https://manned.org/pdbedit>.
+
+- List all Samba users (use verbose flag to shows their settings):
```
Comment (nicokosi) [inline pages/linux/pdbedit.md:7]:
```suggestion
- List all Samba users (use verbose flag to show their settings):
```

File: pages/linux/smbpasswd.md [hunk 6]
```diff
@@ -1,20 +1,21 @@
 # smbpasswd
 
-> Change a user's SMB password.
-> Samba users must also have a local Unix account.
+> Add/remove a Samba user or change their SMB password.
```
Comment (nicokosi) [inline pages/linux/smbpasswd.md:3]: What about?
```suggestion
> Add/remove a Samba user or change its password.
```

**Comment by cyqsimon** [comment]
Quick question: is there anything else required of me? Or am I all set?

File: pages/linux/pdbedit.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+> For simple user add/remove/password, you can also use `smbpasswd`.
+> More information: <https://manned.org/pdbedit>.
+
+- List all Samba users (use verbose flag to show their settings):
+
+`sudo pdbedit --list --verbose`
+
+- Add an existing Unix user to Samba (will prompt for password):
+
+`sudo pdbedit -u {{username}} --create`
```
Comment (nicokosi) [inline pages/linux/pdbedit.md:13]: I suggest sticking on long options everywhere:
```suggestion
`sudo pdbedit --user {{username}} --create`
```

File: pages/linux/pdbedit.md [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+
+- Remove a Samba user:
+
+`sudo pdbedit -u {{username}} --delete`
```
Comment (nicokosi) [inline pages/linux/pdbedit.md:17]: I suggest sticking on long options everywhere:
```suggestion
`sudo pdbedit --user {{username}} --delete`
```

File: pages/linux/pdbedit.md [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+
+- Reset a Samba user's failed password counter:
+
+`sudo pdbedit -u {{username}} --bad-password-count-reset`
```
Comment (nicokosi) [inline pages/linux/pdbedit.md:21]: I suggest sticking on long options everywhere :innocent: :
```suggestion
`sudo pdbedit --user {{username}} --bad-password-count-reset`
```

File: pages/linux/smbpasswd.md [hunk 10]
```diff
@@ -1,20 +1,21 @@
...
 - Modify an existing Samba user's password:
 
-`smbpasswd {{username}}`
+`sudo smbpasswd {{username}}`
 
-- Delete a Samba user:
+- Delete a Samba user (use `pdbedit` instead if the Unix account has been deleted):
 
-`smbpasswd -x {{username}}`
+`sudo smbpasswd -x {{username}}`
```
Comment (nicokosi) [inline pages/linux/smbpasswd.md:21]: Also use long options?

Comment (cyqsimon) [reply pages/linux/smbpasswd.md:21]: `smbpasswd` only provides short options.

**Review by nicokosi — APPROVED** [review]
Seems OK for me, thanks. :+1:

**Review by mfrw — APPROVED** [review]
LGTM.
Thank you @cyqsimon for your contribution.

**Pull request merged**